js/languagePickerView.js

```diff
@@ -22,6 +22,9 @@ define([
             var data = this.model.toJSON();
             var template = Handlebars.templates[this.constructor.template];
             this.$el.html(template(data));
+            if (_.has(data, '_classes') && !_.isEmpty(data, '_classes')) {
+                this.$el.addClass(data._classes);
```

Comment:
```
this.$el.addClass(data._classes);
````
^ this line on it's own should suffice i think?
you won't need to check if ``_classes`` exists on ``data``, because it'll be undefined otherwise and ``$('selector').addClass(undefined);`` doesn't do anything.

see:
```
var obj = {};
$("body").addClass(obj._classes);
//has no impact on the body tag and also doesn't fail.
```